fxcore/chain.json

```diff
@@ -20,7 +20,7 @@
     "fee_tokens": [
       {
         "denom": "FX",
-        "fixed_min_gas_price": 4200000000000,
+        "fixed_min_gas_price": 4000000000000,
```

Comment:
This minimum is still greater than the low amount

Reply:
4000000000000
400000000000
4200000000000
500000000000

Reply:
corrected